frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/utils.ts

```diff
@@ -60,7 +61,7 @@ export const getTopLevelErrorMessage: GetErrorMessage = (errors) => (taskIndex)
 export const findTask = (
   resourceTasks: PipelineBuilderTaskResources,
   task: PipelineTask,
-  t: TFunction,
+  t: TFunction = i18n.t,
```

Comment:
Temporary fix to adopting `i18n.t` in utilities. Will log something later.

Reply:
Create https://issues.redhat.com/browse/ODC-5778 so this can be officially cleaned up in 4.8 (since it was added in 4.8).